### PR TITLE
Gh 90/basic app settings

### LINKED ISSUE #90: [iOS] Basic app settings screen
Display basic app strings 

- [ ] User can see the app version.
- [ ] User can see the network version (Mainnet, Rinkeby)
- [ ] User can view Terms (Link to https://gnosis-safe.io/terms/)
- [ ] User can view Privacy Policy (Link to https://gnosis-safe.io/privacy/)
- [ ] User can view used licenses (Link to https://gnosis-safe.io/licenses/)
- [ ] Terms, PP and licenses are not hardcoded, but links that open a browser window (exactly how we do it in the current app)

https://app.zeplin.io/project/5e8f27b9963e55bb79b13634/screen/5e8f2d80e6c17420dc5f4002

Screenshot 2020-05-11 at 21 49 32

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
closes #90

### CODE REVIEW

File: Multisig/UI/Settings/BasicAppSettingsView.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    var rowHeight: CGFloat = 60
+
+    var body: some View {
+        List {
+            NavigationLink(destination: FiatCurrenciesView()) {
+                keyValueView(key: "Fiat currency", value: "EUR")
+            }
+            .frame(height: rowHeight)
+
+            BrowseLinkButton(title: "Terms of use", url: App.shared.termOfUseURL)
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/BasicAppSettingsView.swift:21]: I think renaming this to `BrowserLink` will be a better name

File: Multisig/UI/UI Library/BrowseLinkButton.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                Spacer()
+                Image(systemName: "chevron.right")
+                    .font(Font.footnote.bold())
+                    .foregroundColor(Color.systemGray6Light)
+            }
+        }
+        .sheet(isPresented: $showsLink, content: browseSafeAddress)
+    }
+
+    func browseSafeAddress() -> some View {
```
Comment (DmitryBespalov) [inline Multisig/UI/UI Library/BrowseLinkButton.swift:38]: this one-line function does not add much value here, I would inline the contents with the line 35.

File: Multisig/UI/UI Library/keyValueView.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+//  keyValueView.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct keyValueView: View {
```
Comment (DmitryBespalov) [inline Multisig/UI/UI Library/keyValueView.swift:11]: typo: should be `KeyValueView`, not `keyValueView`

File: Multisig/UI/Settings/BasicAppSettingsView.swift [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+import SwiftUI
+
+struct BasicAppSettingsView: View {
+    var rowHeight: CGFloat = 60
+
+    var body: some View {
+        List {
+            NavigationLink(destination: FiatCurrenciesView()) {
+                keyValueView(key: "Fiat currency", value: "EUR")
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/BasicAppSettingsView.swift:17]: here it should be "USD"

File: Multisig/Cross-layer/App.swift [hunk 5]
```diff
@@ -27,6 +27,18 @@ class App {
     logger: LogService.shared)
 
     var nodeService = EthereumNodeService(url: URL(string: "https://rinkeby.infura.io/v3/438e11915f8b4834a05e7810b88db4b3")!)
+
+    let termOfUseURL = URL(string:"https://gnosis-safe.io/terms/")!
+    let privacyPolicyURL = URL(string:"https://gnosis-safe.io/privacy/")!
+    let licensesURL = URL(string:"https://gnosis-safe.io/licenses/")!
+    let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? ""
```
Comment (DmitryBespalov) [inline Multisig/Cross-layer/App.swift:34]: I would make the version string of the format "CFBundleShortVersionString (CFBundleVersion)"

Comment (DmitryBespalov) [reply Multisig/Cross-layer/App.swift:34]: like 0.1.0 (2)

File: Multisig/UI/UI Library/BrowseLinkButton.swift [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+//  BrowseLinkButton.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct BrowseLinkButton: View {
```
Comment (DmitryBespalov) [inline Multisig/UI/UI Library/BrowseLinkButton.swift:11]: I like it.

File: Multisig/Cross-layer/App.swift (hunk 5 above)
Comment (DmitryBespalov) [reply Multisig/Cross-layer/App.swift:34]: This comment was not addressed

File: Multisig/UI/Settings/Get In Touch/GetInTouchView.swift [hunk 7]
```diff
@@ -10,75 +10,33 @@ import SwiftUI
...
+            EmailLink(title: "E-mail", url: URL(string:"safe@gnosis.io")!, iconName: "ico-eMail")
 
-                Text("").alert(item: $error) { msg in
-                    Alert(title: Text(msg.value))
-                }
-
-                Text("").sheet(item: $browserURL) { url in
-                    SafariViewController(url: url.value)
-                }
+            ForEach(channels, id: \.url) { item in
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/Get In Touch/GetInTouchView.swift:29]: sweet!

**Comment by MouazAlzahabi** [comment]
closes #90

**Pull request merged**